Longford.md

```diff
@@ -0,0 +1,7 @@
+## Longford
+
+Longford town boasts a state-of-the-art 200-seat theatre, and a four-screen multiplex cinema, with restaurants.
```

Comment:
This sounds amazing @acorkery. Is there a special showing I should try to attend?